int32 id
int32 gesture_no
---
bool gesture_nograb
